release-notes.md

```diff
@@ -1,3 +1,17 @@
+# Release 1.12.1
+
+## What's New
+* The ziti tunnel service is changed to ziti-edge-tunnel service. This service uses the 0.17.x t-sdk version, so the new capabilities like srv support that are added to the tunnel sdk will be available in ziti-edge-tunnel service.
```

Comment:
are there any other things that are going to be new to add here? calling out srv support is cool but probably shouldn't be mentioned until it works/is tested (i don't think it's tested yet?)

Reply:
I will remove srv from this comment